### PR TITLE
Add HTTPs redirects e2e test for RXLB Ingress

### PR DESCRIPTION
**Pull request by panslava**
- Add test implementation, almost a full copy of the TestHttpsRedirects
- Add to regional utils handling https redirect map, which was not expected for ILB (which was the only user of regional utils)

/assign @spencerhance

### CODE REVIEW

File: pkg/fuzz/gcp.go [hunk 1]
```diff
@@ -793,6 +794,15 @@ func RegionalGCLBForVIP(ctx context.Context, c cloud.Cloud, gclb *GCLB, params *
 			if urlMapKey == nil {
 				urlMapKey = urlMapResID.Key
 			}
+			// Ignore redirect urlmaps since they will not have backends, but add them to the gclb map
```
Comment (spencerhance) [inline pkg/fuzz/gcp.go:797]: Let's discuss this offline - I don't think we should be using the redirect map with regional XLB.

Comment (panslava) [reply pkg/fuzz/gcp.go:797]: Redirect map will be created only if user specified HttpsRedirectConfig in FrontendConfig (to redirect Http to https)

Comment (spencerhance) [reply pkg/fuzz/gcp.go:797]: Ah I thought RXLB didn't require this anymore, but I still see it being suggested: https://cloud.google.com/load-balancing/docs/https/setting-up-reg-http-https-redirect

nevermind

**Pull request merged**